src/ucp/core/ucp_ep.h

```diff
@@ -501,6 +502,11 @@ struct ucp_wireup_sockaddr_data {
 } UCS_S_PACKED;
 
 
+struct ucp_wireup_user_data {
+    uint64_t                client_id;      /**< Endpoint client ID */
+};
```

Comment:
1. why not use typedef struct{} here?
2. should it be PACKED since it's passed on the wire?